prac_06/car.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        """Return a string representation of car object."""
+        return f"{self.name}, {self.fuel}, {self._odometer}"
+
```

Comment:
Current __str__ method works, however for usability I would suggest adding in more descriptors to the strings being outputted

i.e. f"Car {self.name}, Fuel = {self.fuel}  ... "